test/crypto/pycurl_test.py

```diff
@@ -0,0 +1,107 @@
+# coding=utf-8
+from __future__ import absolute_import, division, print_function, \
+  unicode_literals
+
+from unittest import TestCase
+
+from iota import TryteString
+from iota.crypto.pycurl import Curl
```

Comment:
Could you change this to `from iota.crypto import Curl`?  That way, I can repurpose these tests for the C extension.

More info:
- https://github.com/iotaledger/iota.lib.py/blob/2.0.2/iota/crypto/__init__.py#L6-L13
- https://github.com/todofixthis/pyota-ccurl/blob/1.0.9/.travis.yml#L12-L17